libmscore/instrchange.cpp

```diff
@@ -104,6 +204,8 @@ void InstrumentChange::write(XmlWriter& xml) const
       {
       xml.stag(this);
       _instrument->write(xml, part());
+      if (_init)
+            xml.tag("init", _init);
```

Comment:
When an instrument change is created, it is automatically given an instrument, even if it hasn't been assigned one by the user yet. And from what I could tell there wasn't a way to tell if an instrument change had been assigned an instrument by the user or not, given the user could theoretically choose the default instrument. Because the copy/paste implementation uses the drop function, which I have implemented to show the instrument change dialogue whenever used, if an instrument change was pasted, it would open the dialogue again, even if it had already been assigned an instrument. And because copying an element uses the xml read/write function to clone the object, rather than a copy, it's necessary to have it in the read/write functions.

Reply:
That makes sense. Thank you. Could you consider summarising that and making it a comment above `_init` in the header file?